src/journal-remote/journal-gatewayd.c

```diff
@@ -1040,7 +1040,7 @@ int main(int argc, char *argv[]) {
                 int flags =
                         MHD_USE_DEBUG |
                         MHD_USE_DUAL_STACK |
-                        MHD_USE_PIPE_FOR_SHUTDOWN |
+                        MHD_USE_ITC |
```

Comment:
The `configure.ac` script demands `libmicrohttpd` *>= 0.9.33*. The problem is, that e.g. *0.9.33* does not provide the `MHD_USE_ITC` flag. 

`MHD_USE_ITC` was introduced with [this commit](http://lists.gnu.org/archive/html/gnunet-svn/2016-10/msg00063.html) so if we change to `MHD_USE_ITC` we must bump the `libmicrohttpd` version in the configure script :)